tests/telemetry/test_writer.py

```diff
@@ -145,37 +145,37 @@ def test_app_started_event_configuration_override(test_agent_session, ddtrace_ru
     configuration = [
```

Comment:
This will fail on python 3.6 and 2.7. We might need to sort this list (similar to how we do it in the app configurations changed test)

Reply:
@mabdinur just updated the test and it's now passed.